### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by Anduin2017**
# Migration

* The main code repo was migrated from Azure DevOps to GitHub.

# New features

* Send file button has a new icon.
* Allow muting a group conversation.
* Update term link and require agreement on the register.
* Better navigation logic when the user presses the go back button on Android.
* Display a warning message when the user tries to sign out.

# Bugs fix

* Fix an issue that user may create groups with an empty name.
* Close sweet alert when page goes back

### CODE REVIEW

File: src/app/Views/register.html [hunk 1]
```diff
@@ -20,6 +20,9 @@ <h1 i18n="@@Kahla">Kahla</h1>
                 <div class="alert" [hidden]="passwordInput.valid || passwordInput.pristine">Password length should between 6 and 100.</div>
                 <input [(ngModel)]="confirmPassword" id="confirmInput" name="confirmPassword" minlength="6" maxlength="100" type="password" i18n-placeholder="@@ConfirmPassword" placeholder="Confirm password" autocomplete="new-password" (keyup.enter)="register()" (ngModelChange)="checkSame()" required #confirmInput="ngModel">
                 <div class="alert" [hidden]="samePassword || confirmInput.pristine">Passwords should be equal.</div>
+                <p>
+                    By clicking "Register" below, you agree to our <a href="https://www.aiursoft.com/docs/terms" target="_blank" rel="noopener noreferrer">terms of service</a> and <a href="https://www.aiursoft.com/docs/terms" target="_blank" rel="noopener noreferrer">privacy statment</a>.
```
Comment (l2dy) [inline src/app/Views/register.html:24]: Why would you add `rel="noopener noreferrer"`?

Comment (xxyzz) [reply src/app/Views/register.html:24]: Just a habit...  https://mathiasbynens.github.io/rel-noopener/

**Review by xxyzz — APPROVED** [review]

**Review by j178 — APPROVED** [review]
LGTM

**Review by ltm0203 — COMMENTED** [review]
假装 审核了。

**Review by ltm0203 — APPROVED** [review]

**Comment by BigSnow1** [comment]
我也假装一下，审核了~

**Comment by BigSnow1** [comment]
我也假装一下，审核了~

**Comment by Anduin2017** [comment]
@BigSnow1 Why close this pull request...

**Review by chunfz — APPROVED** [review]

**Pull request merged**